support-frontend/assets/pages/supporter-plus-landing/twoStepPages/contributionsOnlyLanding.tsx

```diff
@@ -0,0 +1,221 @@
+import { AmountsCard } from '../components/amountsCard';
+
+const recurringContainer = css`
+	background-color: ${palette.brand[400]};
+	border-bottom: 1px solid ${palette.brand[600]};
+	> div {
+		padding: ${space[2]}px 10px ${space[4]}px;
+	}
+	${from.mobileLandscape} {
+		> div {
+			padding: ${space[2]}px ${space[5]}px ${space[4]}px;
+		}
+	}
+	${from.tablet} {
+		border-bottom: none;
+		> div {
+			padding: ${space[2]}px 10px ${space[4]}px;
+		}
+	}
+	${from.desktop} {
+		> div {
+			padding: 40px 10px 72px;
+		}
+	}
+`;
+
+const innerContentContainer = css`
+	max-width: 940px;
+	margin: 0 auto;
+	text-align: center;
+`;
+
+const heading = css`
+	text-align: left;
+	color: ${palette.neutral[100]};
+	${headlineBold24}
+	${from.tablet} {
+		text-align: center;
+	}
+	${from.desktop} {
+		font-size: 2.625rem;
+	}
+`;
+
+const standFirst = css`
+	text-align: left;
+	color: ${palette.neutral[100]};
+	margin: 0 0 ${space[4]}px;
+	${textSans17};
+	line-height: 1.35;
+	strong {
+		font-weight: bold;
+	}
+	${from.tablet} {
+		text-align: center;
+		width: 65%;
+		margin: 0 auto;
+	}
+	${from.desktop} {
+		margin: ${space[4]}px auto ${space[9]}px;
+	}
+`;
+
+const paymentFrequencyButtonsCss = css`
+	margin: ${space[4]}px auto 32px;
+	${from.desktop} {
+		margin: 0 auto ${space[9]}px;
+	}
+`;
+
+const disclaimerContainer = css`
+	background-color: ${palette.brand[400]};
+	> div {
+		border-bottom: 1px solid ${palette.brand[600]};
+		padding: ${space[4]}px 10px;
+	}
+	${from.mobileLandscape} {
+		> div {
+			padding: ${space[5]}px ${space[5]}px;
+		}
+	}
+`;
+
+const links = [
+	{
+		href: 'https://www.theguardian.com/info/privacy',
+		text: 'Privacy policy',
+		isExternal: true,
+	},
+	{
+		text: 'Privacy settings',
+		onClick: () => {
+			cmp.showPrivacyManager();
+		},
+	},
+	{
+		href: 'https://www.theguardian.com/help/contact-us',
+		text: 'Contact us',
+		isExternal: true,
+	},
+	{
+		href: 'https://www.theguardian.com/help',
+		text: 'Help centre',
+		isExternal: true,
+	},
+];
+
+const paymentFrequencyMap = {
+	ONE_OFF: 'One-time',
+	MONTHLY: 'Monthly',
+	ANNUAL: 'Annual',
+};
+
+type ContributionsOnlyLandingProps = {
+	geoId: GeoId;
+};
+export function ContributionsOnlyLanding({
+	geoId,
+}: ContributionsOnlyLandingProps): JSX.Element {
+	const { currencyKey: currencyId, countryGroupId } = getGeoIdConfig(geoId);
+	const countryId = Country.detect();
+
+	const [contributionType, setContributionType] =
+		useState<ContributionType>('MONTHLY');
+
+	const paymentFrequencies: ContributionType[] = [
+		'ONE_OFF',
+		'MONTHLY',
+		'ANNUAL',
+	];
+
+	const handlePaymentFrequencyBtnClick = (buttonIndex: number) => {
+		setContributionType(paymentFrequencies[buttonIndex] as ContributionType);
+	};
+
+	const { selectedAmountsVariant: amounts } = getAmountsTestVariant(
+		countryId,
+		countryGroupId,
+		window.guardian.settings,
+	);
+
+	return (
+		<PageScaffold
+			header={
+				<>
+					<Header>
+						{/* <CountrySwitcherContainer>
```

Comment:
Probably safe to assume we won't have a country switcher, and to remove this

Reply:
Yep, done